lte/gateway/configs/templates/spgw.conf.template

```diff
@@ -46,6 +46,7 @@ S-GW :
       UPLINK_PORT_NUM                      = {{ ovs_uplink_port_number }};
       UPLINK_MAC                           = "{{ ovs_uplink_mac }}";
       MULTI_TUNNEL                         = "{{ ovs_multi_tunnel }}";
+      PIPELINED_CONFIG_ENABLED             = "{{ pipelined_managed_tbl0 }}";
```

Comment:
Sorry one thing I didn't noticed on my last review, where is this value being copied from? I think you still need to add the config generation logic (which is done on `generate_oai_config.py` file)

Reply:
comments addressed.